src/plugins/kibana_usage_collection/server/collectors/kibana/get_saved_object_counts.test.ts

```diff
@@ -16,14 +16,34 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-
+import { elasticsearchServiceMock } from '../../../../../../src/core/server/mocks';
 import { getSavedObjectsCounts } from './get_saved_object_counts';
 
+export function mockGetSavedObjectsCounts(params: any) {
+  const esClient = elasticsearchServiceMock.createClusterClient().asInternalUser;
+  esClient.search
+    // @ts-ignore we only care about the response body
+    .mockResolvedValue(
+      // @ts-ignore we only care about the response body
+      {
+        body: { ...params },
+      }
+    );
+  return esClient;
+}
+
 describe('getSavedObjectsCounts', () => {
   test('Get all the saved objects equal to 0 because no results were found', async () => {
-    const callCluster = jest.fn(() => ({}));
+    const esClient = mockGetSavedObjectsCounts({
+      dashboard: { total: 0 },
+      visualization: { total: 0 },
+      search: { total: 0 },
+      index_pattern: { total: 0 },
+      graph_workspace: { total: 0 },
+      timelion_sheet: { total: 0 },
+    });
```

Comment:
I think this test is checking the behaviour when no results are found:
```suggestion
    const esClient = mockGetSavedObjectsCounts({});
```